src/officehours_api/views.py

```diff
@@ -324,23 +329,22 @@ def get(self, request, queue_id=None, format=None):
         response = HttpResponse(content_type='text/csv')
         response['Content-Disposition'] = f'attachment; filename="{filename}"'
 
+        logger.info(f"User {username} successfully exported meeting start logs for queues {repr(queues_user_is_in)}.")
+        self.extract_log(queues_user_is_in, response)
+
+        return response
+
+    @staticmethod
+    def extract_log(queues, response):
         writer = csv.writer(response)
         with connection.cursor() as cursor:
-            # Generate a string of placeholders, one for each item in the list
-            queue_id_placeholders = ', '.join(['%s'] * len(queues_user_is_in))
+            queue_ids = ', '.join(map(str, queues))
 
-            # Add a query to just get the logs for the queues_user_is_in
-            cursor.execute(f"SELECT * FROM meeting_start_logs where queue_id in ({queue_id_placeholders})", queues_user_is_in)
+            cursor.execute(
+                'SELECT * FROM meeting_start_logs '
+                f'where queue_id in ({queue_ids})')
             rows = cursor.fetchall()
 
-            # Get column names from the cursor description
-            column_names = [desc[0] for desc in cursor.description]
-
-            # Write the headers
-            writer.writerow(column_names)
+            column_names = map(lambda λ: λ[0], cursor.description)
```

Comment:
I was reviewing through this and don't know why this change was made unless it was just trying to be cute or done by AI. What was there before was fine and this just makes it less maintainable by introducing a unicode character into a function as a variable.